auth_session_timeout/__openerp__.py

```diff
@@ -9,7 +9,8 @@
     'summary': """
         This module disable all inactive sessions since a given delay""",
 
-    'author': "ACSONE SA/NV, Dhinesh D, Odoo Community Association (OCA)",
+    'author': "ACSONE SA/NV, Dhinesh D, Jesse Morgan, \
+Odoo Community Association (OCA)",
```

Comment:
Used an escaped newline to fix issue with new author string length.
Mangles the file layout somewhat.
Wasn't sure if un-escaped newlines using `"""` would be legal in the author string.